### ISSUE TITLE
New command to create a new configuration from a template

### ISSUE THREAD

**Issue by jjpaulo2** [jjpaulo2/pipconf#4]
We can add a new command to the tool, to create a configuration file from a template, to reduce the time necessary to create a new configuration.

e.g.: `pipconf new <config-name>`

The template must contain every parameters listten in the oficial pip documentation.

https://pip.pypa.io/en/stable/topics/configuration/#configuration-files

---
It must be interesting to follow the same that pylint did. The command `pylint --generate-toml-config` sends to stdout a content of .toml file with all configuration parameters possible in pylint configuration.

```toml
[tool.pylint.main]
# Analyse import fallback blocks. This can be used to support both Python 2 and 3
# compatible code, which means that the block might have code that exists only in
# one or another interpreter, leading to false positives when analysed.
# analyse-fallback-blocks =

# Clear in-memory caches upon conclusion of linting. Useful if running pylint in
# a server-like mode.
# clear-cache-post-run =
...
```

**Comment by jjpaulo2** [comment]
@sheory I think it would be a nice issue to work now. We can keep in contact or talk through a pull request.

**Comment by jjpaulo2** [comment]
@sheory all the package have many descartable comments. When I did it, I thought it was a good idea, but today, with some clean code knowledge, I see that it is bad.

Feel free to delete any unecessary comment.

**Comment by jjpaulo2** [comment]
Solved by #12.
